tests/env/env.go

```diff
@@ -488,3 +488,8 @@ func (e *TestEnv) TearDown() {
 
 	glog.Infof("finish tearing down...")
 }
+
+func NewBackendAuthTestEnv(port uint16) *TestEnv {
```

Comment:
Nit: Should this be here? I don't think `env.go` should reference a specific test name

Do we even need this function? All it does is call another function anyways with 1 hardcoded argument